main.py

```diff
@@ -1,10 +1,54 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_dict = json.load(players_file).items()
+
+    for player, player_attr in players_dict:
+
+        if not Race.objects.filter(name=player_attr["race"]["name"]).exists():
+            Race.objects.create(
```

Comment:
Use .get_or_create https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create

Reply:
can i use it to all create instead .exists?